metron-deployment/packaging/docker/rpm-docker/SPECS/metron.spec

```diff
@@ -317,6 +316,8 @@ This package installs the Metron Profiler %{metron_home}
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 %changelog
+* Thu Jan 19 2017 Justin Leet <justinjleet@gmail.com> - 0.3.0
```

Comment:
I'd just yank that section or leave it blank if it's required. No point in figuring out how to keep it if we're agreed we don't need it.